shap/plots/_force.py

```diff
@@ -90,15 +117,19 @@ def force(base_value, shap_values=None, features=None, feature_names=None, out_n
         elif len(base_value) > 1 and np.all(base_value == base_value[0]):
             base_value = base_value[0]
 
-    if isinstance(base_value, np.ndarray) or isinstance(base_value, list):
+    if isinstance(base_value, (np.ndarray, list)):
         if not isinstance(shap_values, list) or len(shap_values) != len(base_value):
-            raise Exception("In v0.20 force_plot now requires the base value as the first parameter! " \
-                            "Try shap.force_plot(explainer.expected_value, shap_values) or " \
-                            "for multi-output models try " \
-                            "shap.force_plot(explainer.expected_value[0], shap_values[0]).")
-
+            emsg = (
+                "In v0.20, force plot now requires the base value as the first parameter! "
+                "Try shap.plots.force(explainer.expected_value, shap_values) or "
+                "for multi-output models try "
+                "shap.plots.force(explainer.expected_value[0], shap_values[0])."
+            )
+            raise Exception(emsg)
```

Comment:
Would a TypeError be more appropriate here?

Reply:
Changed to `TypeError`. I also used `DimensionError` on the only other `Exception` call in `shap/plots/_force.py`.